### PR TITLE
Replace depedency securemock with mockito.

### PR DESCRIPTION
**Pull request by mkleen**
## Checklist

 - [X] User relevant changes are recorded in ``CHANGES.txt``
 - [ ] Touched code is covered by tests
 - [ ] Documentation has been updated if necessary
 - [X] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by seut** [comment]
@mkleen Please check test failures.

**Comment by mkleen** [comment]
> @mkleen Please check test failures.

Seems to be now all working.

File: sql/src/test/java/io/crate/execution/engine/collect/files/FileReadingCollectorTest.java [hunk 1]
```diff
@@ -196,14 +195,13 @@ public void testCollectWithEmptyLine() throws Throwable {
     public void testCollectWithOneSocketTimeout() throws Throwable {
         S3ObjectInputStream inputStream = mock(S3ObjectInputStream.class);
 
-        when(inputStream.read(new byte[anyInt()], anyInt(), anyByte()))
+        when(inputStream.read(any(), anyInt(), anyInt()))
```
Comment (seut) [inline sql/src/test/java/io/crate/execution/engine/collect/files/FileReadingCollectorTest.java]: afaik you could also use `any(byte[].class)`

File: sql/src/test/java/io/crate/execution/jobs/CountTaskTest.java [hunk 2]
```diff
@@ -78,7 +78,7 @@ public void testClose() throws Exception {
 
         // on error
         future = new CompletableFuture<>();
-        when(countOperation.count(eq(txnCtx), anyMap(), any(Symbol.class))).thenReturn(future);
+        when(countOperation.count(eq(txnCtx), any(), any(Symbol.class))).thenReturn(future);
```
Comment (seut) [inline sql/src/test/java/io/crate/execution/jobs/CountTaskTest.java]: just curious: why isn't `anyMap` working here? (also I think there is now an unused import ;)

File: sql/src/test/java/io/crate/execution/engine/collect/files/FileReadingCollectorTest.java (hunk 1 above)
Comment (mkleen) [reply sql/src/test/java/io/crate/execution/engine/collect/files/FileReadingCollectorTest.java]: it actually does.

**Pull request merged**